public/__tests__/kibi_timeline.js

```diff
@@ -138,6 +147,330 @@ describe('KibiTimeline Directive', function () {
     });
   });
 
+  it('correct filter should be created - (default) ID one', function (done) {
+    initTimeline({
+      startField: '@timestamp',
+      endField: '',
+      labelField: 'machine.os'
+    });
+
+    const date = '25-01-1995';
+    const dateObj = moment(date, 'DD-MM-YYYY');
+    const results = {
+      took: 73,
+      timed_out: false,
+      _shards: {
+        total: 144,
+        successful: 144,
+        failed: 0
+      },
+      hits: {
+        total : 49487,
+        max_score : 1.0,
+        hits: [
+          {
+            _index: 'logstash-2014.09.09',
+            _type: 'apache',
+            _id: '61',
+            _score: 1,
+            _source: {
+              '@timestamp': date,
+              machine: {
+                os: 'linux'
+              }
+            },
+            fields: {
+              '@timestamp': [ dateObj ]
+            }
+          }
+        ]
+      }
+    };
+    const addFilterSpy = sinon.spy(queryFilter, 'addFilters');
+
+    searchSource.crankResults(results);
+    $scope.$digest();
+
+    const expectedFilters = [{
+      query: {
+        ids: {
+          type: 'apache',
+          values: ['61']
+        }
+      },
+      meta: {
+        index: 'logstash-*'
+      }
+    }];
+
+    // wait until timeline is fully rendered
+    setTimeout(function () {
```

Comment:
it would be better to avoid setTimeout since on some machine the timeline might not be rendered after 200ms.
can you use some `complete` event for that ?